frontend/src/storage/exampleExpeditionData.json

```diff
@@ -46,7 +46,7 @@
             "difficulty": "Średnie",
             "options": ["Tak", "Nie"],
             "points": 20,
-            "next": [3, 4],
+            "next": [3, 4, -2],
```

Comment:
```suggestion
            "next": [-2],
```

Błąd który został tutaj od pierwszej wersji JSONa, można go od razu złapać.

Reply:
Faktycznie, skoro to już koniec grafu to powinno być tylko id końca. Za chwilę poprawię